lib/ret/api/token_utils.ex

```diff
@@ -20,4 +22,96 @@ defmodule Ret.Api.TokenUtils do
       account_id: account.account_id
     })
   end
+
+  defp ensure_atom(x) when is_atom(x), do: x
+  defp ensure_atom(x) when is_binary(x), do: String.to_atom(x)
+
+  # TODO Should we permit the creation of tokens
+  # on behalf of other accounts like this?
```

Comment:
It feels odd to allow this, but it seemed like an easy way for us to do a closed alpha/beta test of the API on HMC. Alternatively, we could add a column (e.g. an integer bitmask) to the database and use it to assign roles, then check these roles to see if the given account has permission to create credentials.